enhancements/ptp-ha.md

```diff
@@ -0,0 +1,82 @@
+---
+title: ptp_multi-nic_ha_support
+authors:
+  - "@danielmellado"
+reviewers:
+  - "@zshi-redhat"
+  - "@SchSeba"
+  - "@aneeshkp"
+  - "@imiller0"
+approvers:
+  - "@ijolliffe"
+creation-date: 2022-06-15
+last-updated: 2022-06-15
+tracking-link: TBD
+status: provisional
+---
+
+# PTP multi-nic HA Support
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Multi-nic support for PTP operator [1] seems to be already built-in. In
+enterprise telco environments, there's often a need to load balance two
+different signals from different PTP IEEE 1588-2008 sources.
+
+PTP uses best master clock (BMC) algorithm to perform a selection of the
+candidate clockb based on a series of parameters such as accuracy, quality,
+stratum and uuid. The HA support should also consider these ones and perform
+the change when needed.
+
+## Motivation
+
+### Goals
+
+- Support multiple PTP sources within the PTP operator
+- Add an additional field in the CR regarding stratum priority
+
+### Non-Goals
+
+## Proposal
+
+Would this be a new controller on the operator? Some new parameter on the
+reconcile loop? Opening this EP so we can iterate on this.
+
```

Comment:
Yeah, that was my idea! A new  controller within the ptp operator. Basically, what I was wondering is whether this would need to be a new full fledged controller or just a new field/change within the existing one ;)

Reply:
ok please add to the document so that is captured as the direction in the review.